### PR TITLE
Updates to deploy options for kind

### PR DESCRIPTION
**Pull request by ghchinoy**
* Modifies kind config to expose 80 (instead of 8080) and 443
* adds in a kuard-httpproxy.yaml for an example in sites/example hierarchy
* adds in some clarity to the "Test with HTTPProxy" section to reference the new kuard-httpproxy.yaml

reference #1858

### CODE REVIEW

**Comment by stevesloka** [comment]
Hey @ghchinoy could you fix the build error? 

```
error    no new line character at the end of file  (new-line-at-end-of-file)
```

**Comment by stevesloka** [comment]
Sorry, @ghchinoy I merged a big docs update and now there's a conflict. Would you mind one last pass and fixing that as well as merging your commits into a single one? Thanks!

File: site/examples/kuard-httpproxy.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```
Comment (davecheney) [inline site/examples/kuard-httpproxy.yaml]: Where is this file referenced from?

File: examples/kind/kind-expose-port.yaml [hunk 2]
```diff
@@ -4,6 +4,10 @@ nodes:
 - role: control-plane
 - role: worker
   extraPortMappings:
-  - containerPort: 8080
-    hostPort: 8080
+  - containerPort: 80
```
Comment (davecheney) [inline examples/kind/kind-expose-port.yaml]: Were the original 8080 ports incorrect? As far as I understand it Contour configures the Envoy containers to listen on 8080 and 8443.

Comment (ghchinoy) [reply examples/kind/kind-expose-port.yaml]: No, the 8080 ports worked ok; I'd changed to 80 to align with blog post examples and other documentations.
Would be happy to change back to 8080 and add 8443 and redo docs for consistency.

File: site/examples/kuard-httpproxy.yaml (hunk 1 above)
Comment (ghchinoy) [reply site/examples/kuard-httpproxy.yaml]: Should be site/docs/master/deploy-options.md ... which appears to be missing from the latest squash. Oops (will correct).

Comment (davecheney) [reply site/examples/kuard-httpproxy.yaml]: Ping. Would you please rebase and squash to one commit then reply so we know to review again. Thanks Dave

**Comment by davecheney** [comment]
Thank you for working on this and I want to apologise for not having the full context here. 

Fundamentally the problem I have with this change is we spent a lot of time reducing the number of examples we include with the project to one. I’m concerned that this change increases the number of different deployment examples that we include with the product. 

Would you be able to step back a little and outline the problems with the current documentation — accepting of course that there are simple things like missing liked files. 

Ideally I’d like to resolve #1858 without adding a second deployment example.

**Comment by ghchinoy** [comment]
@davecheney  this makes a lot of sense. I suggest that this PR be closed and a short discussion either in channel or on #1858 occur to clarify and align blog posts and in-repo examples.

**Pull request closed without merging**